### DISCUSSION TITLE
Unable to access codespace.

### DISCUSSION THREAD

**Discussion by jkugsiya** [community/community#30755] [Codespaces]
I am having a github pro account and I cannot see the codespace code button on any of my repositories. If this feature is in beta, can I get beta signup form link so that I can get started with it.
Thanks

**Comment by alexagillman** [comment]
Hi, you should have access to Codespaces if this conditions listed [here](https://docs.github.com/en/codespaces/developing-in-codespaces/creating-a-codespace#access-to-github-codespaces) are met
